middleware/error_handler.js

```diff
@@ -0,0 +1,6 @@
+import logger from '../config/logger/logger-service.js';
+
+export default (err, req, res, next) => {
+    res.status(500).send('Oops!');
+    logger.warn("Error in controller: ", err);
```

Comment:
it's better to log error before sending a response.

Reply:
switched lines